### ISSUE TITLE
Ребят вы там что-то компелируете.....

### ISSUE THREAD

**Issue by MokaSka** [Maxx53/SCMBot#551]
блин ну ребят скиньте норм версию программы или пошагово объясните как вы делаете чтоб покупалось,,, ведь на 90% уверен что тут не каждый програмист, создайте тогда уж нормальную версию и киньте архивом мне плз

**Comment by init92** [comment]
да, давайте тебе скинем, потом наши редлайны будешь скупать все

**Comment by immorro** [comment]
@MokaSka , https://github.com/Maxx53/SCMBot/issues/549#issuecomment-160313449

**Comment by MokaSka** [comment]
@immorro да и ладно, танков мало, девушка...., да хрен с ней, бот то лучше

**Comment by PaRaDoX1KZ** [comment]
@immorro почему-то в твоей версии даже логиниться не хочет(((

**Comment by immorro** [comment]
@PaRaDoX1KZ , логин работает, как я понимаю, только через копирование куки. Обычный способ умер с введением аутентификатора

**Comment by PaRaDoX1KZ** [comment]
@immorro я и так копировал куки из браузера....

**Comment by immorro** [comment]
@PaRaDoX1KZ , ну тогда хз... я там одну строчку поменял и она никак не могла дропнуть логин. Как вариант, можешь вкинуть exe в папку, где у тебя старая версия работала.

**Comment by PaRaDoX1KZ** [comment]
@immorro уже пробовал)) терь и старая версия слетела) хз почему)
